CHANGELOG.md

```diff
@@ -1,35 +1,106 @@
 # nf-core/rnaseq: Changelog
 
-## Version 1.3
+## Version 1.4dev
+
+### Pipeline updates
+
+* Add option `--skipTrimming` to be able to run from previous processed data since this step take quite long.
+* Restore `SummarizedExperimment` object creation in the salmon_merge process avoiding increaasing memory with sampe size.
+* Add option `--skipBiotypeQC` to be able to skip BiotypeQC in `featureCounts` if e.g. the utilized GTF/GFF doesn't contain the required information
+* Adjust wording of skipped samples [in pipeline output](https://github.com/nf-core/rnaseq/issues/290)
+* Added [possibility to output unmapped reads](https://github.com/nf-core/rnaseq/issues/277) to pipeline
+* Add option to modify `featureCounts -t` option to assign reads to a different type than `exon`. This makes it compatible with nuclearRNAseq. For instance, it would count reads in introns and exons if `gene` is used instead. Nuclear RNA has un-spliced RNA, and the whole transcript, including the introns, need to be counted, e.g. by specifying `--fc_count_type transcript`.
+* Fix sample names in feature counts and dupRadar to remove suffixes added in other processes
+* Removed `genebody_coverage` process [#195](https://github.com/nf-core/rnaseq/issues/195)
+* Implemented Pearsons correlation instead of euclidean distance [#146](https://github.com/nf-core/rnaseq/issues/146)
+* Add `--stringTieIgnoreGTF` parameter [#206](https://github.com/nf-core/rnaseq/issues/206)
+* Resolved link to guidelines is broken [#203](https://github.com/nf-core/rnaseq/issues/203)
+* Removed unnecessary `stringtie` channels for `MultiQC`
+* Added tximport to merge salmon output
+* Added Salmon as an supplementary method to STAR and HiSAT2
```

Comment:
We could do with tidying up this changelog quite a bit - maybe move some of these to the start, with a summary of the major changes first?

Reply:
Summary added, added some headings, too to get this a bit easier to follow and read. Let me know what you think, added all of this to `https://github.com/nf-core/rnaseq/tree/mini-release-fixes`, so that others in the team can adjust / help on this too 👍